src/www/app/cordova_main.ts

```diff
@@ -15,6 +15,8 @@
 /// <reference path='../../types/ambient/outlinePlugin.d.ts'/>
 /// <reference path='../../types/ambient/webintents.d.ts'/>
 
+import '../ui_components/app-root.js';
```

Comment:
Move this to main instead. There's no reference to app root in this file.

Reply:
Done.